src/components/TxDetailsModal.js

```diff
@@ -105,18 +103,6 @@ class TxDetailsModal extends Component {
                   {isNc && <ListItem title={t`Blueprint Method`} text={ncMethod} />}
                   {isNc && <ListItem title={t`Nano Contract ID`} text={ncIdComponent} />}
                   {isNc && <ListItem title={t`Nano Contract Caller`} text={ncCallerAddrComponent} />}
-                  {isNc
-                    && (
-                      <ListItem
-                        title={t`Nano Contract Status`}
-                        text={(
-                          <TransactionStatusLabel
-                            hasFirstBlock={hasFirstBlock}
-                            isVoided={isVoided}
-                          />
-                        )}
-                      />
-                    )}
```

Comment:
I know I said to remove the status because it won't matter if the tx is confirmed by a block for Hathor Play but I think we should keep a status if the tx is voided, it seems important for the wallet user to know if the tx was invalidated somehow

Reply:
Ok. I've added the status label only for voided tx: 37c14d0c29141a94c9959900587235ccd0b1c68c